pkg/endpoint/endpoint.go

```diff
@@ -2742,3 +2748,17 @@ func (e *Endpoint) scrubIPsInConntrackTable() {
 	e.scrubIPsInConntrackTableLocked()
 	e.Unlock()
 }
+
+func (e *Endpoint) GetUUIDLocked() string {
+	if e.uuid == "" {
+		e.uuid = uuid.NewUUID().String()
```

Comment:
Because you are generating the NewUUID based on something random. I would prefer the "real" uuid to be something like `fmt.Sprintf("%s-%s-%s", ClusterName, NodeName, EndpointID)`

Reply:
endpoint ID is not unique over time. The same endpoint ID can be reused. What's wrong with using a guaranteed unique UUID for these purposes?